atecc608a_se.c

```diff
@@ -60,7 +60,7 @@
 
 static ATCAIfaceCfg atca_iface_config = {
     .iface_type = ATCA_I2C_IFACE,
-    .devtype = ATECC608A,
+    .devtype = ATECC508A,
```

Comment:
Why was this change necessary? Explain the motivation in the commit message body. It's unclear if there was anything broken or misused by having this be 608A.